### DISCUSSION TITLE
error in compiling with git clone

### DISCUSSION THREAD

**Discussion by chlsu** [mitmproxy/mitmproxy#6323] [Q&A] [answered]
I've downloaded from https://github.com/conorpp/MiTM-HTTP-Proxy/tree/master
through git clone.

but there is some error in compiling with make.

clang-16: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [src/proxy/makefile:15: all] Error 1
make[1]: Leaving directory '/root/working/mitmproxy/MiTM-HTTP-Proxy'
make: *** [makefile:32: src/proxy/build/proxy.so] Error 2

can I have some advice?

**Comment by Prinzhorn** [comment]
This is an entirely unrelated project, how did you end up here?

**Reply by chlsu** [reply]
i need C code because of project. I could not proceed anymore, after error. Is this code not related to mitmproxy or is it not working inherently?

**Reply by Prinzhorn** [reply, marked as answer]
> Is this code not related to mitmproxy

Not at all
